tools/cli/wskaction.py

```diff
@@ -170,7 +174,10 @@ def getExec(self, args, props):
             exe = self.getActionExec(args, props, pipeAction)
         elif args.artifact is not None and os.path.isfile(args.artifact):
             contents = open(args.artifact, 'rb').read()
-            if args.artifact.endswith('.swift'):
+            if args.kind in ['swift:3','swift:3.0','swift:3.0.0']:
```

Comment:
The question is basically about consistency between the REST API (where currently creating a Swift3 action means having a field `"kind": "swift3"` in the POST payload) and the CLI, where currently creating a Swift3 action means using `--kind=swift:3`. They're inconsistent.